test/postsubmit-tests-with-pipeline-deployment.sh

```diff
@@ -114,14 +114,16 @@ elif [[ ${CLOUDBUILD_FINISHED} == TIMEOUT ]];then
   exit 1
 fi
 
-# Deploy Kubeflow
-source "${DIR}/deploy-kubeflow.sh"
+COMMIT_SHA=$PULL_BASE_SHA source "${DIR}/deploy-cluster.sh"
```

Comment:
the old code seems just uses pull_pull_sha. 
https://github.com/kubeflow/pipelines/blob/2e7f2d48160dfb9b6911afcc244e434adf7410e4/test/deploy-kubeflow.sh#L20
just to double check, is this pull base sha the right SHA?

Reply:
I think it might be a problem of previous code (this also works if PULL_PULL_SHA is empty).
https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#job-environment-variables
when running postsubmit test, there's only PULL_BASE_SHA, there's no PULL_PULL_SHA

Reply:
That explains the empty sha in the GKE cluster I saw before. Great catch!